pkg/pyramid/pyramid.go

```diff
@@ -14,14 +14,14 @@ import (
 type FS interface {
 	// Create creates a new file in the FS.
 	// It will only be persistent after the returned file is stored.
-	Create(ctx context.Context, namespace string) (StoredFile, error)
+	Create(ctx context.Context, storageID string, namespace string) (StoredFile, error)
 
 	// Open finds the referenced file and returns its read-only File.
 	// If file isn't in the local disk, it is fetched from the block storage.
-	Open(ctx context.Context, namespace, filename string) (File, error)
+	Open(ctx context.Context, storageID string, namespace, filename string) (File, error)
 
 	// Exists returns true if filename currently exists on block storage.
-	Exists(ctx context.Context, namespace, filename string) (bool, error)
+	Exists(ctx context.Context, storageID string, namespace, filename string) (bool, error)
 
 	// GetRemoteURI returns the URI for filename on block storage.  That URI might not
```

Comment:
Seems like `GetRemoteURI` is not using `namespace` - 
Can you validate if that's really the case, and if so, remove it?

Reply:
See [this comment](https://github.com/treeverse/lakeFS/pull/8549/files#r1935141318).